pdfcpu.yaml

```
@@ -0,0 +1,24 @@
+package:
+  name: pdfcpu
+  version: 0.8.1
+  epoch: 0
+  description: A Go PDF processor and CLI
+  copyright:
+    - license: Apache-2.0
+
+pipeline:
+  - uses: go/install
```

can we switch to go/build instead and ad the necessary build instructions here 
```suggestion
  - uses: go/build
```

---

Sure, I'll swap it shortly.